src/vsr/superblock_fuzz.zig

```diff
@@ -310,14 +315,29 @@ const Environment = struct {
             .view = env.superblock.staging.vsr_state.view + 5,
         };
 
+        var vsr_headers = vsr.ViewChangeHeaders.BoundedArray{ .buffer = undefined };
```

Comment:
```suggestion
        var vsr_headers = vsr.ViewChangeHeaders.BoundedArray.init(0);
```

Reply:
I avoid `BoundedArray.init()` because it returns an error based on the length passed in, so it would need to be `catch unreachable`.